README.md

```diff
@@ -1,3 +1,5 @@
-# Northcoders News API
+# NC News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Setting up Environments:
+
+Add database environment to `.env` files in order to switch between test and development environments accordingly. The files are ignored from uploading to github for security.
```

Comment:
I would be more explicit in what someone cloning this will need to do. 

 - make two new files with stated names 
 - populate those files with provided environment variables

Reply:
thanks